### ISSUE TITLE
consulting?

### ISSUE THREAD

**Issue by mpinkus** [vgolovanov/OpenThreadDotNet#4]
are you available to consult on a poc using this library and nanoframework?

**Comment by vgolovanov** [comment]
Hi.
It depends on the time, but I'm available for consulting and glad to help you.

**Comment by mpinkus** [comment]
I’d like to do a poc using the stm32wb5m discovery kit… I’d like to get a 4+ board test mesh environment running using mani framework… what would this take and how much of your time? Estimated cost? Once I have poc of message passing and device onboarding we can take it from their. I currently have 10 kits FYI.

Mike Pinkus
Mobile
________________________________

**Comment by mpinkus** [comment]
Btw, Jose has indicated he can add this kit to targets in parallel in a week.

Mike Pinkus
Mobile
________________________________

**Comment by vgolovanov** [comment]
Development repo moved to a new location and latest source code available at
https://github.com/openthreaddotnet/nanoFramework.OpenThread. You can use the stm32wb5m board only as an NCP coprocessor and if NCP firmware is available from ST. But you still need a host MCU which is supported by nanoFramework.

**Comment by mpinkus** [comment]
Yes we will use another stm cortex for that. Could this be done in 20 hours? Poc? Board setup etc?

Mike Pinkus
Mobile
________________________________

**Comment by vgolovanov** [comment]
Yes. All setup can be done in a few hours. What messenger is better for you ?

**Comment by mpinkus** [comment]
I can add you to a slack channel … what would this cost for 20 hours? Can you send me an estimate to my email? I’d like to start tomorrow if possible … do you have this dev kit available to you? I can fedex a couple if not.

Send email address for me to invite you?

Mike

**Comment by vgolovanov** [comment]
I can't find your email.
